source/features/hide-inactive-deployments.tsx

```diff
@@ -10,7 +10,6 @@ function init(): void {
 	deployments.pop(); // Don't hide the last deployment, even if it is inactive
 
 	for (const deployment of deployments) {
-		// TODO: Rewrite with :has selector, CSS-only feature
 		if (select.exists('[title="Deployment Status Label: Inactive"]', deployment)) {
 			deployment.remove();
```

Comment:
I don’t think it's possible because the logic is:

1. Take all the deployments
2. Exclude the last one
3. Hide inactive

The second step is impossible in CSS selectors.

Reply:
did you mean the third step is impossible with just CSS selectors?

(because the code referenced is about the 3rd step)

Reply:
The code I referenced is everything above this line. It's the second part that's not possible, because selectors don't have a concept of "how many items this selectors selects". [jQuery did](https://api.jquery.com/last-selector/), CSS doesn't (`.deployments:not(:last)`)